pkgs/by-name/an/ankama-launcher/package.nix

```
@@ -0,0 +1,44 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}:
+let
+  pname = "ankama-launcher";
+  version = "3.12.17";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://launcher.cdn.ankama.com/installers/production/Ankama%20Launcher-Setup-x86_64.AppImage";
+    hash = "sha256-aD5YYcye/WPpb0Kir55qCL+RGbK8ZSdmi4NKmqKkW/Y=";
```

Hash needs to be updated.
```suggestion
        hash = "sha256-JERXkIp0psOu5L3bSpuOgfxokZ1pB4/BHpL2np2aHWo=";
```